cellprofiler/modules/graytocolor.py

```diff
@@ -76,94 +81,122 @@ def create_settings(self):
         # # # # # # # # # # # # # # # #
         self.red_image_name = cps.ImageNameSubscriber(
                 "Select the image to be colored red",
-                can_be_blank=True, blank_text=LEAVE_THIS_BLACK)
+                can_be_blank=True, blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image to be displayed in red""")
 
         self.green_image_name = cps.ImageNameSubscriber(
                 "Select the image to be colored green",
-                can_be_blank=True, blank_text=LEAVE_THIS_BLACK)
+                can_be_blank=True, blank_text=LEAVE_THIS_BLACK,
+                doc="""Select the input image to be displayed in green""")
```

Comment:
Please add `*(Used only if "%(SCHEME_RGB)s" is selected as the color scheme)*` plus a newline at the beginning of the doc string.  Sorry, should have thought of this before!